src/services/rns/models/expiration.model.ts

```diff
@@ -0,0 +1,12 @@
+import { Column, DataType, Model, Table, HasMany, BelongsTo, ForeignKey } from 'sequelize-typescript'
+import Domain from './domain.model'
+
+
+@Table({ freezeTableName: true, tableName: 'rns_domain_expiration', timestamps: false })
+export default class DomainExpiration extends Model {
```

Comment:
Hmmmm 😅

Is this new Model the solution for the "synchronization problem"?

Because from the database point of view I don't like this design in the same way as I didn't like the `OwnerAddress` model proposition you had earlier 😅

Reply:
Hmm re-read the PR's description, so found an answer to the question.

I see, alright :-)